controller/config/beerocks_controller.conf.in

```diff
@@ -21,7 +21,6 @@ load_client_band_steering=1
 load_client_optimal_path_roaming=1
 load_legacy_client_roaming=1
 load_client_11k_roaming=1
-load_steer_on_vaps=@BEEROCKS_WLAN1_STEERING_VAPS@,@BEEROCKS_WLAN2_STEERING_VAPS@
```

Comment:
Please squash to the previous commit.

Reply:
I intentionally separated the commits - since this commit is part of the `controller` and the previous one is part of the `common`